packages/reaction-core/server/main.js

```diff
@@ -23,6 +23,17 @@ _.extend(ReactionCore, {
     ReactionRegistry.createDefaultAdminUser();
     // hook after init finished
     ReactionCore.Hooks.Events.run("afterCoreInit", this);
+
+    // Make sure this event is registered last.
+    ReactionCore.Hooks.Events.add("orders/sendNotification", (result) => {
+      if (!result) {
+        //  If event was successfully captured the result will be falsy.
+        //  No-op
+      } else {
+         ReactionCore.Log.warn("Package email-notifications seems not being " +
```

Comment:
you're correct. The package is installed by default. This is just intended as safety net, which is not necessary at all. Feel free to remove it if you don't feel it adds value.